lib/python/gladevcp/iconview.py

```diff
@@ -102,7 +102,7 @@ def __init__(self):
         # This will hold the path we will return
         self.path = ""
         self.button_state = {}
-        self.old_button_state = {}        
+        self.old_button_state = {}
```

Comment:
in the future, it doesn't hurt to make whitespace-tidying changes a separate commit.  It's nice to see the cruft cleaned up in any case.